charts/testkube-operator/values.yaml

```diff
@@ -29,6 +29,47 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+webhook:
+  enabled: true
+  name: testkube-operator-webhook-admission
+  labels: {}
+  annotations: {}
+
+  patch:
+    enabled: true
+    image:
+      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+      tag: v1.3.0
+      pullPolicy: IfNotPresent
+    annotations: {}
+    podAnnotations: {}
+    labels: {}
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 2000
```

Comment:
why 2000, when do we create it?

Reply:
That is what NGINX Controller uses for the kube-webhook-certgen and they have their own fork of the image, and as I used their version of kube-webhook-certgen, I copied the same securityContext settings

Reply:
ok, then it's fine